package/kernel/r8125/patches/0002-print-link-speed-and-duplex-mode.patch

```diff
@@ -0,0 +1,37 @@
+From 6b89b2bb75fd12195631b8009c0548d1d81bec99 Mon Sep 17 00:00:00 2001
+From: Chukun Pan <amadeus@jmu.edu.cn>
+Date: Sun, 4 Aug 2024 21:16:23 +0800
+Subject: [PATCH] r8125: print link speed and duplex mode
+
+Like other Ethernet drivers, print link speed and duplex mode
+when the interface is up. Formatting output at the same time.
+
+Signed-off-by: Chukun Pan <amadeus@jmu.edu.cn>
+---
+ src/r8125_n.c | 8 ++++++--
+ 1 file changed, 6 insertions(+), 2 deletions(-)
+
+--- a/src/r8125_n.c
++++ b/src/r8125_n.c
+@@ -5116,15 +5116,19 @@ static void
+ _rtl8125_check_link_status(struct net_device *dev)
+ {
+         struct rtl8125_private *tp = netdev_priv(dev);
++        u16 status = RTL_R16(tp, PHYstatus);
```

Comment:
AFAIK this could be moved under `if (netif_msg_ifup(tp))` because we're only using it there.